include/crm/common/results.h

```diff
@@ -363,6 +363,7 @@ enum ocf_exitcode pcmk_rc2ocf(int rc);
 int pcmk_rc2legacy(int rc);
 int pcmk_legacy2rc(int legacy_rc);
 int pcmk_gaierror2rc(int gai);
+int pcmk_bz2error2rc(int bz2);
```

Comment:
This can also be internal. To avoid the 2...2, let's call it bzlib2rc, since the library uses "bzlib" (e.g. bzlib.h)

Reply:
Renamed and moved to results_internal.h.